docs/security/user-certs.md

```diff
@@ -19,11 +20,30 @@ When you request a certificate your provide some public personal information abo
 
 Your **name** and **email** address are encoded into the signed certificate and **if either of those change** (usually email address) then you should **request a new certificate** with the correct information and **revoke the old certificate**.
 
-### Requesting a User Certificate
+### Getting a Certificate from CILogon
 
-There are two different ways to obtain your certificate, either via a command line interface, or through a web browser. The links below will guide you through whichever method you choose. The command line method is generally simpler and less prone to errors than using a web browser but it may require extra setup and package installation the first time you do it. Most people end up needing their certificate for both command line grid use and web browser use and it is generally easier to import your certificate into your browser (and email client) from the command line certificate files than to export your certificate from your web browser to use on the command line. This is because of the large variety of web browsers each with it's own way to deal with certificates. If you will use the certificate ONLY in your web browser then requesting it from your browser is probably the easiest method.
+You will have to obtain your user certificate using the [CILogon web UI](https://cilogon.org/).
+Follow the steps below to get an user certificate:
 
-[Get or renew a certificate with command line interface.](../security/certificate-management.md)
+1. First, either search for your institution and select it or scroll through list and do the same. <br>![Institution Selection](/img/cilogon_select_idp.png)
+1. Click the `Log On` button and enter your instutional credentials.
+1. Upon successfully entering your credentials, you'll get a page asking for you to enter a password.  Enter a password that is at least 12 characters long and then click on the `Get New Certificate` button.<br>![Password entry](/img/cilogon_cert_password.png) 
+1. The web page will generate a `usercred.p12` file and prompt you to download it.  The certiticate will be protected
+using the password you entered in the prior step.
+
+### Getting a Certificate from the OSG CA
+
+There are two different ways to obtain your certificate, either via a command line interface, or through a web browser.
+The links below will guide you through whichever method you choose.
+The command line method is generally simpler and less prone to errors than using a web browser but it may require extra
+setup and package installation the first time you do it.
+Most people end up needing their certificate for both command line grid use and web browser use and it is generally
+easier to import your certificate into your browser (and email client) from the command line certificate files than to
+export your certificate from your web browser to use on the command line. 
+This is because of the large variety of web browsers each with it's own way to deal with certificates. If you will use
```

Comment:
>exporting your certificate from your web browser is hard because there are lots of other browsers too

It sounds like the thing that is hard is not exporting your certificate from your browser, but the OSG documenting all possible ways any user might want to do this... :-x